tests/unit/models/package-info-cache-test.js

```diff
@@ -0,0 +1,219 @@
+'use strict';
+
+const path = require('path');
+const expect = require('chai').expect;
+const Project = require('../../../lib/models/project');
+let addonFixturePath = path.resolve(__dirname, '../../fixtures/addon');
+const MockUI = require('console-ui/mock');
+const MockCLI = require('../../helpers/mock-cli');
+
+describe('models/package-info-cache.js', function() {
+  let project, projectPath, packageJsonPath, packageContents, projectPackageInfo, ui, cli, pic;
+  this.timeout(100000);
```

Comment:
this timeout seems rather high, the tests bellow all seem super fast. I suspect this was only added to aide local debugging, but I believe it can be removed

Reply:
The original timeout was 500000! I'll reset down to 20000, which is still about 4x what's really necessary.